### PR TITLE
Africa Covid Watch Version One 

### PR DESCRIPTION
**Pull request by Benmuiruri**
### React Capstone Project
**Dev - @Benmuiruri** 

### Tech/Techniques used

- ES6 JavaScript.
- React
- React Bootstrap
- CSS
- callbacks and promises.
- Webpack.
- AAA pattern for unit tests.
- Write unit tests for a React app.
- Send and receive data from an API.
- Use AddContext to manage user logged in status
- Use local storage to store user logged in status
- Use API documentation.
- Here's a round-up of the features that were implemented in this project

### API Used
Covid Vaccine dATA API provided by [M-Media-Group](https://github.com/M-Media-Group/Covid-19-API)

### Website Features

- Login page 
- A home page showing a covid vaccine data for all African countries 
- unique route to each country's individual data.
- Input filter to search a particular country. 

### Link to video presentation
[Loom Video](https://www.loom.com/share/94a38dc88ecb471d8e5a69aaff040040)

### CODE REVIEW

File: src/components/Country/CountriesList.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
```
Comment (elmar8287) [inline src/components/Country/CountriesList.js:1]: - [ ] Please, consider that you have 1 test failed
Screen Shot 2022-05-20 at 16 38 56
Screen Shot 2022-05-20 at 16 39 47

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
- Nice design
- Informative presentation video
- Nice implementation of project requerements

## Required Changes ♻️

- [ ]  _Check the comments under the review._

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Country/CountriesList.js (hunk 1 above)
Comment (Benmuiruri) [reply src/components/Country/CountriesList.js:1]: Hi Elmar, I will look into it, see what might be causing the issue.

**Review by brytebee — APPROVED** [review]
Hi, @Benmuiruri & @elmar8287👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Benmuiruri** [comment]
Thanks for the review @brytebee  and @elmar8287

**Pull request merged**